src/lib/containers/UserCard.tsx

```diff
@@ -18,14 +19,14 @@ export type UserCardSize =
 export type UserCardProps = {
   /** A UserProfile may be used for data for the card. You must supply one of `userProfile`, `alias`, `ownerId` */
   userProfile?: UserProfile
-  /** Whether or not to hide the user's Synapse email address */
-  hideEmail?: boolean
-  /** If set, the corresponding image will be shown for the user. */
-  preSignedURL?: string
   /** An alias that resolves the ownerId for the UserProfile. You must supply one of `userProfile`, `alias`, `ownerId` */
   alias?: string
   /** The unique ownerId of the UserProfile. You must supply one of `userProfile`, `alias`, `ownerId` */
   ownerId?: string
+  /** Whether or not to hide the user's Synapse email address */
+  hideEmail?: boolean
+  /** If set, the corresponding image will be shown for the user. */
+  preSignedURL?: string
```

Comment:
Just reordered so userProfile, alias, and ownerId are all next to each other, because one of those is needed.